apps/site-landing/lang/en.json

```diff
@@ -1,428 +1,428 @@
 {
-  "'+yxUhg'": {
```

Comment:
Not sure where those "'" are coming from.

Reply:
I think I found them.  There were quotes in the package.json (check last commit) that seem to be treated differently in win/iOS